.evergreen/config.yml

```diff
@@ -843,10 +847,14 @@ axes:
         variables:
            TOX_ENV: "asyncio-py39"
            PYTHON_BINARY: "/opt/python/3.9/bin/python3"
+      - id: "asyncio-py310"
+        variables:
+           TOX_ENV: "asyncio-py310"
+           PYTHON_BINARY: "/opt/python/3.10/bin/python3"
       - id: "py3-pymongo-latest"
         variables:
            TOX_ENV: "py3-pymongo-latest"
-           PYTHON_BINARY: "/opt/python/3.7/bin/python3"
```

Comment:
Is this supposed to be the min version?

Reply:
Agreed. I think we should keep this as the min version we support for now. This test fails on 3.10:
```
 [2022/10/25 17:32:56.873] ImportError: Failed to import test module: tornado_tests
 [2022/10/25 17:32:56.873] Traceback (most recent call last):
 [2022/10/25 17:32:56.873]   File "/opt/python/3.10/lib/python3.10/unittest/loader.py", line 470, in _find_test_path
 [2022/10/25 17:32:56.873]     package = self._get_module_from_name(name)
 [2022/10/25 17:32:56.873]   File "/data/mci/593377d0ad434cdc1766d60588588194/src/test/__init__.py", line 72, in _get_module_from_name
 [2022/10/25 17:32:56.873]     return super()._get_module_from_name(name)
 [2022/10/25 17:32:56.873]   File "/opt/python/3.10/lib/python3.10/unittest/loader.py", line 377, in _get_module_from_name
 [2022/10/25 17:32:56.873]     __import__(name)
 [2022/10/25 17:32:56.873]   File "/data/mci/593377d0ad434cdc1766d60588588194/src/test/tornado_tests/__init__.py", line 25, in <module>
 [2022/10/25 17:32:56.873]     from tornado import testing
 [2022/10/25 17:32:56.873]   File "/data/mci/593377d0ad434cdc1766d60588588194/src/.tox/py3-pymongo-latest/lib/python3.10/site-packages/tornado/testing.py", line 16, in <module>
 [2022/10/25 17:32:56.873]     from tornado.httpclient import AsyncHTTPClient
 [2022/10/25 17:32:56.873]   File "/data/mci/593377d0ad434cdc1766d60588588194/src/.tox/py3-pymongo-latest/lib/python3.10/site-packages/tornado/httpclient.py", line 50, in <module>
 [2022/10/25 17:32:56.873]     from tornado import gen, httputil, stack_context
 [2022/10/25 17:32:56.873]   File "/data/mci/593377d0ad434cdc1766d60588588194/src/.tox/py3-pymongo-latest/lib/python3.10/site-packages/tornado/httputil.py", line 107, in <module>
 [2022/10/25 17:32:56.873]     class HTTPHeaders(collections.MutableMapping):
 [2022/10/25 17:32:56.873] AttributeError: module 'collections' has no attribute 'MutableMapping'
```

Reply:
Fixed